packages/twenty-server/src/engine/core-modules/serverless/drivers/local.driver.ts

```diff
@@ -167,7 +200,6 @@ export class LocalDriver
           });
         }
         child.kill();
-        fs.unlink(tmpFilePath).catch(console.error);
```

Comment:
no, we do not deleting files because we need them each time we update the function
Your question leads to the network and data performance issues we will probably need to look at in some time
But currently, when we update the serverless function code, we update the code in the file storage, then we download the code from the file storage to the server temporary folder, then we transpile the js. It is required as we transpile a code base now, and not only a string like before